static/css/devhub/base.less

```diff
@@ -1,32 +1,45 @@
 @import '../common/normalize.css';
 @import '../impala/fonts/fira/fira.less';
 
-// Colors
-@primary-font: 'Fira Sans', sans-serif;
-@primary-font-color: #fff;
-@secondary-font-color: #333;
-@tertiary-font-color: #40d38d;
-
-@button-border: #979797;
-@primary-button-background: #29c978;
-
-@primary-section-background: #0568ba;
-@secondary-section-background: #f3f5f8;
-@tertiary-section-background: #1d3770;
-
+// Grid
+@grid-max: 940px;
 @phone-large: ~"only screen and (min-width: 428px)";
 @phone: ~"only screen and (max-width: 768px)";
```

Comment:
yeah, I was talking about defining media-queries on a case-by-case and on a content bases. E.g

```css
div {
    @media (max-width: 768px) {
        ...
    }
}
```

feels, at least the bigger the css gets, more reasonable compared to defining all these to variables. You'll easily end up with bootstrap-style `x-sm-l-foobar` (imaginative) definitions. Although I do agree that naming things makes it easier to have more specific breakpoints and makes everything behave the same-ish on viewport resizing.